.circleci/config.yml

```diff
@@ -132,10 +132,19 @@ jobs:
           environment:
             KUBECONFIG: /home/circleci/project/data/kubeconfig
             GO111MODULE: "on"
+      - run:
+          when: always
```

Comment:
Should this be `when: on_dail` instead?

Reply:
I theory vpp crash may not cause test failure. 
Either way this is a very cheap operation.